index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- avatar component -->
+    <div class = "comp" id = "avatar-component">
+        <h1 class = "heading1">Avatar</h1>
+        <p class = "component-info">Avatars are graphical representation of users or user persona</p>    
+        <img class = "first-avatar" src="avatar1.jpg" alt="">
```

Comment:
Do not keep the alt attribute of an image as empty and for better performance you can attach a loading="lazy" attribute to the image itself